### ISSUE TITLE
Wide spell list information is misaligned

### ISSUE THREAD

**Issue by tasonir** [jeremygurr/crawl_spicy#12]
<img width="1680" alt="screen shot 2018-03-08 at 12 29 11 pm" src="https://user-images.githubusercontent.com/3357531/37174775-699be708-22cc-11e8-881c-f02e24fc8774.png">

The fail % shows up under level, etc.  It's one tab too far to the right.

**Comment by jeremygurr** [comment]
Yeah I've been fighting with this one for a while. If I set the width to what it seems like it should be, it goes too far the other way.
